react-native/react/screen-orientation.mo.md

```diff
@@ -0,0 +1,54 @@
+# [MO] Handle screen orientation (iOS & Android)
+## Owner: [Julien Nassar](https://github.com/juliennassar)
+## Sponsor: [Alexandre Moureaux](https://github.com/almouro)
+
+> **Note**: Please create an [issue](https://github.com/bamlab/dev-standards/issues/new) or even a Pull Request with your feedback, typo correction.
+
+## Context
+
+You may sometimes need to control the screen orientation of your app. You might want to fix your screen orientation to portrait mode only, or, depending on the page you are in, you might want to specify a screen orientation for some of your screens.
```

Comment:
do you have example of changing orientation for some screens?
I can think of UGC: changing orientation for full screen videos only